--- origsrc/goom2k4-0/configure.in	2005-02-07 07:46:41.000000000 -0600
+++ src/goom2k4-0/configure.in	2008-05-12 22:55:16.062500000 -0500
@@ -49,10 +49,6 @@
   	CCAS='$(CC)'
   	AC_SUBST(CCAS)
 	;;
-*-*-cygwin*)
-  	CFLAGS="$CFLAGS -mno-cygwin -mwindows"
-	LDFLAGS="$LDFLAGS -lmingw32"
-	;;
 esac
 
 dnl ARCH
--- origsrc/goom2k4-0/sdl-goom/Makefile.am	2005-02-07 07:46:41.000000000 -0600
+++ src/goom2k4-0/sdl-goom/Makefile.am	2008-05-12 23:26:06.218750000 -0500
@@ -6,6 +6,6 @@
 goom2_LDADD = $(top_builddir)/src/libgoom2.la
 goom2_LDFLAGS = $(SDL_LIBS) `gtk-config --libs`
 goom2_SOURCES = frame_rate_tester.c gmtimer.c gtk-callbacks.c gtk-interface.c gtk-support.c pixeldoubler.c readme.c sdl_goom.c sdl_pixeldoubler.c surface.c
-INCLUDES=$(SDL_CFLAGS) `gtk-config --cflags` -I../src/
+INCLUDES=$(SDL_CFLAGS) `gtk-config --cflags` -I$(top_srcdir)/src/
 endif
 
--- origsrc/goom2k4-0/src/Makefile.am	2005-02-07 07:46:41.000000000 -0600
+++ src/goom2k4-0/src/Makefile.am	2008-05-12 22:47:07.125000000 -0500
@@ -17,7 +17,7 @@
 
 goom2_library_includedir=$(includedir)/goom
 goom2_library_include_HEADERS = goom.h goom_plugin_info.h goom_typedefs.h goom_graphic.h goom_config_param.h goom_visual_fx.h goom_filters.h goom_tools.h goomsl.h goomsl_hash.h goomsl_heap.h goom_tools.h goom_config.h
-libgoom2_la_LDFLAGS = -export-dynamic -export-symbols-regex "goom.*" 
+libgoom2_la_LDFLAGS = -export-dynamic -export-symbols-regex "goom.*" -no-undefined
 libgoom2_la_SOURCES = \
 	goomsl_yacc.y goomsl_lex.l goomsl.c goomsl_hash.c goomsl_heap.c \
 	goom_tools.c $(MMX_FILES) $(PPC_FILES) \
--- origsrc/goom2k4-0/xmms-goom/Makefile.am	2005-02-07 07:46:42.000000000 -0600
+++ src/goom2k4-0/xmms-goom/Makefile.am	2008-05-12 23:25:51.968750000 -0500
@@ -5,7 +5,7 @@
 xmms_libdir = $(XMMS_VISUALIZATION_PLUGIN_DIR)
 libxmmsgoom2_la_LIBADD = $(top_builddir)/src/libgoom2.la $(XMMS_LIBS)
 libxmmsgoom2_la_SOURCES = xmms_goom.c
-INCLUDES=-DDATADIR=\"@XMMS_DATA_DIR@\" @XMMS_CFLAGS@ -Wall -I../src/
+INCLUDES=-DDATADIR=\"@XMMS_DATA_DIR@\" @XMMS_CFLAGS@ -Wall -I$(top_srcdir)/src/
 endif
 
-libxmmsgoom2_la_LDFLAGS = -module -avoid-version
+libxmmsgoom2_la_LDFLAGS = -module -avoid-version -no-undefined
